app/javascript/add-another.js

```diff
@@ -85,7 +85,7 @@ const setUpSections = (sectionList) => {
 const setNumbering = (section, counter) => {
   const counterElement = section.querySelector('[data-add-another-role="counter"]')
   if (counterElement) {
-    counterElement.innerHTML = counterElement.dataset.addAnotherNumberFrom === "zero" ? counter : counter + 1;
+    counterElement.innerHTML = counter + 1;
```

Comment:
question - so the result will still be `: counter 1`, `: counter 2`, `: counter 3`, but simplified as there is no longer this (HTML label?) `data-add-another-number-from="zero"`?

Reply:
So previously, the default was for the sequence  of counters was `1`, `2`,  3`, but if `data-add-another-number-from="zero"` was set it would go `0` `1` `2` (this was used for additional bank accounts, where the first one wouldn't have a label, then the second would be called `additional bank account 1` and the third would be `additional bank account 2`).

Now, for bank accounts, as with everything else, the first is called `bank account 1`, the second is `bank account 2` the third is `bank account 3`. So now we always start from 1, and never start from `0`, so  we've removed the option to start from zero.

Reply:
That makes sense now. Thank you.